book/publishing/guidelines/guidelines-editors.md

```diff
@@ -0,0 +1,35 @@
+(pb-guidelines-editors)=
+
+# Guide for Notebooks Editors
+Welcome to the guide for notebooks editors of the EDS book! 
+
+Before you begin this process, please be sure to [understand how the publication process works](pb-guidelines).
+
+We have a [code of conduct](https://raw.githubusercontent.com/alan-turing-institute/environmental-ds-book/master/CODE_OF_CONDUCT.md) which is mandatory for everyone involved in the review process of our notebooks.
+
+Below you will find the main steps we suggest to follow as an editor of a notebook to EDS book.
+
+## Scope and Aims
+Thank you for considering managing the review workflow of a notebook submission to EDS book.
+Our review process aims to be open, collaborative, transparent and inclusive. 
+We therefore welcome editors from a diversity of background and with varying levels of programming/domain expertise.
+
+## Pre-review
+Once a notebook idea comes in, it will be validated by Editors-in-chief(EiC). 
```

Comment:
```suggestion
Once a notebook idea comes in, it will be validated by Editors-in-chief (EiC). 
```

Reply:
Fixed in [a8a97e5](https://github.com/alan-turing-institute/environmental-ds-book/pull/117/commits/a8a97e5569253e0f0fb9f7f985c50d9febd7ef4d).